Examples/25_rotor_position_control.py

```diff
@@ -0,0 +1,120 @@
+'''
+----------- 25_rotor_position_control --------------
+Run ROSCO with rotor position control
+-------------------------------------
+
+Run a steady simulation, use the azimuth output as an input to the next steady simulation, with different ICs 
+
+'''
+
+import os, platform
+from ROSCO_toolbox.ofTools.case_gen.run_FAST import run_FAST_ROSCO
+from ROSCO_toolbox.ofTools.case_gen import CaseLibrary as cl
+from ROSCO_toolbox.ofTools.fast_io import output_processing
+from ROSCO_toolbox.controller import OpenLoopControl
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+#directories
+this_dir            = os.path.dirname(os.path.abspath(__file__))
+rosco_dir         = os.path.dirname(this_dir)
+example_out_dir     = os.path.join(this_dir,'examples_out')
+os.makedirs(example_out_dir,exist_ok=True)
+
+if platform.system() == 'Windows':
+    lib_name = os.path.realpath(os.path.join(this_dir, '../ROSCO/build/libdiscon.dll'))
+elif platform.system() == 'Darwin':
+    lib_name = os.path.realpath(os.path.join(this_dir, '../ROSCO/build/libdiscon.dylib'))
+else:
+    lib_name = os.path.realpath(os.path.join(this_dir, '../ROSCO/build/libdiscon.so'))
+
+def main():
```

Comment:
Ran into an error
<img width="959" alt="image" src="https://github.com/NREL/ROSCO/assets/49625841/44bf411e-1110-42f2-b304-9c78c0766bc3">

Reply:
Update docs to use python=3.9, but we might be able to get rid of that exact designation.